### PR TITLE
Feature: Terms and conditions UI

### LINKED ISSUE #2873: Organization registration page: replace tides in shortcode example with Glific
**Describe the bug**
Organization registration page: replace tides in shortcode example with Glific

**Screenshots**
image

### PR DESCRIPTION
**Pull request by akanshaaa19**
resolves #2873 #2875

### CODE REVIEW

**Comment by kurund** [comment]
@akanshaaa19 

I have a few questions on code structure, it might be easier to discuss over the call.

**Comment by akanshaaa19** [comment]
> @akanshaaa19
> 
> I have a few questions on code structure, it might be easier to discuss over the call.

okay, let me know if we can connect today?

File: src/containers/Organization/Onboarding/Steps/OrgDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
+import { Input } from 'components/UI/Form/Input/Input';
+import * as Yup from 'yup';
+import { FormLayout } from '../FormLayout/FormLayout';
+import { useState } from 'react';
+
+import styles from '../FormLayout/FormLayout.module.css';
+import { ONBOARD_URL_UPDATE } from 'config';
+import axios from 'axios';
```
Comment (kurund) [inline src/containers/Organization/Onboarding/Steps/OrgDetails.tsx:8]: Can you please reorder the imports to include external packages first followed by internal ones.

File: src/containers/Organization/Onboarding/Steps/PlatformDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  const [api_key, setApiKeys] = useState<string>('');
+  const [shortcode, setShortcode] = useState<string>('');
+  const [phone, setPhone] = useState<string>('');
+
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const FormSchema = Yup.object().shape({
+    name: Yup.string()
+      .required('Organisation Name is required.')
```
Comment (kurund) [inline src/containers/Organization/Onboarding/Steps/PlatformDetails.tsx:27]: Can you please add localization for the messages?

File: src/containers/Organization/Onboarding/Steps/SigningAuthority.tsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+  const [permissions, setPermissions] = useState({
+    terms_agreed: false,
+    support_staff_account: true,
+  });
+
+  const [loading, setLoading] = useState(false);
+
+  const FormSchema = Yup.object().shape({
+    submitterName: Yup.string()
+      .required(t('Name is required.'))
```
Comment (kurund) [inline src/containers/Organization/Onboarding/Steps/SigningAuthority.tsx:37]: please add localization for the messages.

**Review by kurund — CHANGES_REQUESTED** [review]
Overall code looks good. Please check the comments. I have also pushed minor fixes.

**Comment by kurund** [comment]
@akanshaaa19 

I have also cleaned the form and updated the labels. It would be good to deploy it on staging.

**Review by kurund — CHANGES_REQUESTED** [review]
Can we also make the Reach out form compact, it has lot of whitespace at bottom.

**Comment by kurund** [comment]
@akanshaaa19 

I have also made a few changes to the landing page and shortcode logic. It would be good to deploy the same on staging.

**Comment by akanshaaa19** [comment]
> @akanshaaa19
> 
> I have also made a few changes to the landing page and shortcode logic. It would be good to deploy the same on staging.

made the require changes, its good to go now @kurund

**Comment by kurund** [comment]
@KrishnaDwibhashi2 @akanshaaa19 

I thought about the names of the steps and made a few changes. Please let me know what you think.

Screenshot 2024-05-25 at 17 55 50

I was also wondering if we should change the step order.  Move Organization as the first step and Bot details to the second.

**Comment by KrishnaDwibhashi2** [comment]
@kurund 

I kind of feel it incomplete without the details suffix. Like the word "Billing" instead of "payment", but again having it as Billing details would make it complete for me (this is a very subjective opinion)

Initial design had org name first & then bot details. But as we won't be able to create entry at backend if the bot details are not asked in the first page, we changed the sequence.

**Comment by kurund** [comment]
@KrishnaDwibhashi2 

Let's discuss and finalize this during our call.

**Review by kurund — APPROVED** [review]
Looks good.

**Pull request merged**